src/main/java/de/samply/directory_sync/directory/DirectoryApi.java

```diff
@@ -130,7 +130,8 @@ public Either<OperationOutcome, Biobank> fetchBiobank(BbmriEricId id) {
   }
 
   private HttpGet fetchBiobankRequest(BbmriEricId id) {
-    HttpGet request = new HttpGet(urlPrefix(id.getCountryCode()) + "_biobanks/" + id);
+    HttpGet request = new HttpGet(
+        baseUrl + "/api/v2/eu_bbmri_eric_biobanks/" + id.toString());
```

Comment:
toString() isn't necessary here because the plus operator implicitly calls testing on the second argument